include/zephyr/dt-bindings/dma/rpi_pico_dma.h

```diff
@@ -31,6 +31,8 @@
 #define RPI_PICO_DMA_SLOT_PIO1_RX1   RPI_PICO_DMA_DREQ_TO_SLOT(0x0D)
 #define RPI_PICO_DMA_SLOT_PIO1_RX2   RPI_PICO_DMA_DREQ_TO_SLOT(0x0E)
 #define RPI_PICO_DMA_SLOT_PIO1_RX3   RPI_PICO_DMA_DREQ_TO_SLOT(0x0F)
+
+#if defined(CONFIG_SOC_SERIES_RP2040)
```

Comment:
We cannot use the Kconfig value in devicetree.

Reply:
Correct.

@Manu3l0us I've tweaked your original code a bit compared to your branch. Please take a look and confirm you're happy with the attribution/authorship as it stands.

Reply:
Thank you @ajf58 

This looks good to me.